### DISCUSSION TITLE
Include ESP-CAM Current Voltage Report

### DISCUSSION THREAD

**Discussion by ramon2k10** [jomjol/AI-on-the-edge-device#3142] [General]
### The Feature

Hello, I have implemented the battery-powered system.  For me it would be very appreciated if with each measurement I could receive the current Voltage parameter of the ESP-CAM in the json.

**Comment by caco3** [comment]
You mean the supply power of the ESP-board?
There is no hardware implemented that could measure this!
You would need to build your own hardware. Also note that the used hardware (ESP32-CAM) is not optimized for low power usage!
